cmd/kubelet/app/server.go

```diff
@@ -1015,6 +1015,8 @@ func startKubelet(k kubelet.Bootstrap, podCfg *config.PodConfig, kubeCfg *kubele
 	if kubeCfg.ReadOnlyPort > 0 {
 		go k.ListenAndServeReadOnly(net.ParseIP(kubeCfg.Address), uint(kubeCfg.ReadOnlyPort))
 	}
+
+	go k.ListenAndServePodResources()
```

Comment:
Should this be feature gated?

Reply:
Hmmm... it is an opt-in feature by nature of it being a new API.  @vishh WDYT?

Reply:
+1 to a feature gate

Reply:
added